include/mixer.h

```diff
@@ -96,7 +97,7 @@ extern int16_t lut_u8to16[UINT8_MAX + 1];
 
 static constexpr auto max_filter_order = 16;
 
-static constexpr auto millis_in_second = 1000.0;
+static constexpr auto millis_in_second = 1000.0f;
```

Comment:
Most of the current audio devices use this as a double to convert to a time-value (that's used against PIC times, which are also doubles). 
```c++
src/hardware/disney.cpp:        ms_per_frame = millis_in_second / dss_7khz_rate;
src/hardware/gameblaster.h:     static constexpr auto ms_per_render  = millis_in_second / render_rate_hz;
src/hardware/gus.cpp:constexpr double MS_PER_DMA_XFER = millis_in_second / DMA_TRANSFERS_PER_S;
src/hardware/gus.cpp:   ms_per_render = millis_in_second / audio_channel->GetSampleRate();
src/hardware/gus.cpp:           ms_per_render = millis_in_second / frame_rate_hz;
src/hardware/innovation.cpp:    ms_per_clock = millis_in_second / chip_clock;
src/hardware/lpt_dac.cpp:       ms_per_frame = millis_in_second / channel->GetSampleRate();
src/hardware/opl.cpp:   ms_per_frame = millis_in_second / sample_rate;
src/hardware/pcspeaker_impulse.cpp:     const auto portion_of_ms = static_cast <double>(index) / millis_in_second;
src/hardware/ps1audio.cpp:      static constexpr auto ms_per_render    = millis_in_second / render_rate_hz;
src/hardware/tandy_sound.cpp:   static constexpr auto ms_per_render  = millis_in_second / render_rate_hz;
```

My only thought is something similar to the `exp vs. expf`, `sin vs sinf`: 

``` c++
static constexpr auto millis_in_second = 1000.0;
static constexpr auto millis_in_second_f = 1000.0f;
```

Reply:
Ah, I just blindly changed it... Introducing a float variant then.